services/src/main/java/org/keycloak/protocol/oidc/endpoints/request/AuthorizationEndpointRequestParserProcessor.java

```diff
@@ -64,19 +64,30 @@ public static AuthorizationEndpointRequest parseRequest(EventBuilder event, Keyc
             String requestObjectRequired = OIDCAdvancedConfigWrapper.fromClientModel(client).getRequestObjectRequired();
 
             if (OIDCConfigAttributes.REQUEST_OBJECT_REQUIRED_REQUEST_OR_REQUEST_URI.equals(requestObjectRequired)
-                    && requestParam == null && requestUriParam == null) {
+                        && requestParam == null && requestUriParam == null) {
```

Comment:
Minor: is it possible to remove formatting changes from this class?

Reply:
@mposolda As I've mentioned just above, I will find and revert these points that are not related to this PR.

Reply:
Thanks